src/HeaderCell.ts

```diff
@@ -24,9 +24,13 @@ class HeaderCell extends HeaderCellBase<HeaderCellProperties> {
 			onSortRequest
 		} = this.properties;
 
+		// sortDetail: undefined => 'asc'
```

Comment:
It's not commented out, I was trying to note the different states `sortDetail` could be in and what direction they should result in.